.github/workflows/maven.yml

```diff
@@ -0,0 +1,25 @@
+name: Java CI
```

Comment:
This project is using [Travis](https://travis-ci.org/hkamel/sonar-auth-aad) as a CI (even though its broken currently), why add another?

Reply:
I was debating about ripping out TravisCI, as it's pretty broken for PRs, and Java in general now. Currently, I don't see any way to trigger SonarCloud scans with GitHub Actions/Workflows. That makes it less attractive than just switching to Azure DevOps.